### PR TITLE
Document package copy command with OpenAPI

### PR DESCRIPTION
**Pull request by danidoni**

### CODE REVIEW

File: src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_copy.yaml [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+          - 0
+      description: Setting both makeoriginolder and makeolder will return an error back.
+    - in: query
+      name: nodelay
+      schema:
+        type: string
+        enum:
+          - 1
+          - 0
+      description: Set to '1' to defer the copy to a background job.
```
Comment (hennevogel) [inline src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_copy.yaml:58]:
```suggestion
      description: Set to '1' to do the copying synchronously. By default the copying is done asynchronously.
```

background job is an implementation detail...

File: src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_copy.yaml [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    - in: query
+      name: withhistory
+      schema:
+        type: string
+        enum:
+          - 1
+          - 0
+      description: Set to '1' to copy the package history.
```
Comment (hennevogel) [inline src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_copy.yaml:66]:
```suggestion
      description: Set to '1' to copy the package commit history.
```

File: src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_copy.yaml [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+        type: string
+      description: Old project
+    - in: query
+      name: withbinaries
+      schema:
+        type: string
+        enum:
+          - 1
+          - 0
+      description: Copy binaries also.
```
Comment (hennevogel) [inline src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_copy.yaml:34]:
```suggestion
      description: Copy binaries that have been build by the origin package too.
```

File: src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_copy.yaml [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+          - 1
+          - 0
+      description: Set to '1' to copy the package history.
+    - in: query
+      name: noservice
+      schema:
+        type: string
+        enum:
+          - 1
+          - 0
```
Comment (hennevogel) [inline src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_copy.yaml:73]: 1 disables running services after the copy.

**Review by hennevogel — CHANGES_REQUESTED** [review]

File: src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_copy.yaml [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+      description: Copy binaries also.
+    - in: query
+      name: makeolder
+      schema:
+        type: string
+        enum:
+          - 1
+          - 0
+      description: Setting both makeolder and makeoriginolder will return an error back.
```
[the remark this answers is not in the document]
Comment (danidoni) [reply src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_copy.yaml:42]: I still don't know... :/

[the remark this answers is not in the document]
Comment (danidoni) [reply src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_copy.yaml]: Found it

**Review by hennevogel — APPROVED** [review]

**Pull request merged**